lib/shared/influxdb.rb

```diff
@@ -0,0 +1,188 @@
+require 'puppet/http'
```

Comment:
Let's move this to `puppet_x/puppetlabs/influxdb/influxdb.rb` to be consistent with the Puppet extensions. This allows namespacing the code to avoid conflicts.